sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -465,7 +464,7 @@ class CodegenContext {
  * into generated class.
  */
 abstract class GeneratedClass {
```

Comment:
it'd be great to write a little bit more documentation on what this is. It took me a while to page this in.

I can do it in a different pr.